sql/2020/22_HTTP_2/percent_of_resources_loaded_over_h2_or_h1_1_per_site.sql

```diff
@@ -0,0 +1,28 @@
+# standardSQL
+# Percentage of resources loaded over HTTP/2 or HTTP/1.1 per site
+SELECT
+  client,
+  APPROX_QUANTILES((http_1_1/num_requests) * 100, 1000)[OFFSET(500)] AS median_avg_pct_http_1_1,
+  APPROX_QUANTILES((http_2/num_requests) * 100, 1000)[OFFSET(500)] AS median_avg_pct_http_2
+FROM (
+  SELECT 
+    percentile,
+    client,
+    page,
+    COUNT(0) AS num_requests, 
+    SUM(IF(JSON_EXTRACT_SCALAR(payload, "$._protocol") ="http/0.9",1,0)) AS http_0_9, 
+    SUM(IF(JSON_EXTRACT_SCALAR(payload, "$._protocol") ="http/1.0",1,0)) AS http_1_0, 
+    SUM(IF(JSON_EXTRACT_SCALAR(payload, "$._protocol") ="http/1.1",1,0)) AS http_1_1,
+    SUM(IF(JSON_EXTRACT_SCALAR(payload, "$._protocol") ="HTTP/2",1,0)) AS http_2
```

Comment:
As discussed we should add `h3` (e.g. `h3-Q050`) to this and also check if protocol is case sensitive or not.

Reply:
Hopefully the requests table will get updated and if so then i will be able to drop all of the manual extraction from payload.

In regards to H3 I am not seeing protocol = `h3-Q050` The only entires I am seeing are:

![image](https://user-images.githubusercontent.com/8494683/93408197-f7bb8800-f861-11ea-8f2b-8f39f232b694.png)

IMHO I think if we use:
`SUM(IF(protocol="http/0.9",1,0)) AS http_0_9,
    SUM(IF(protocol="http/1.0",1,0)) AS http_1_0,
    SUM(IF(protocol="http/1.1",1,0)) AS http_1_1,
    SUM(IF(protocol="HTTP/2",1,0)) AS http_2,
    SUM(IF(protocol IN ("QUIC","http/2+quic/46"),1,0)) AS quic`

we should be ok. Thoughts? Perhaps I am missing the intent of the comment...?

Reply:
I added `SUM(IF(JSON_EXTRACT_SCALAR(payload, "$._protocol") IN ("QUIC","http/2+quic/46"),1,0)) AS quic` to the query in order to collect H3